### PR TITLE
fix: add kratos/hydra integration example

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
      vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the
      maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by vinckr** [comment]
Adding your example as discuss @atreya2011, this is the correct branch to link to right? 

fyi @reviewer we will add this example to ory/examples in the near future, wanted to give it exposure right away as it is very helpful.

File: docs/ecosystem/community.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ modifies the core technology. This isn't an exhaustive list. Want your own conte
 - [Starter for Svelte Kit and the Ory Stack](https://github.com/MicLeey/sveltekit-ory-starter)
 - [Reference Ory Docker Compose Setup ](https://github.com/radekg/ory-reference-compose) +
   [Article](https://gruchalski.com/posts/2021-04-10-ory-reference-docker-compose-and-thoughts-on-the-platform/)
-- [WIP Ory Hydra/Kratos Integration in Go](https://github.com/atreya2011/go-kratos-test)
+- [Ory Hydra/Kratos Integration in Go](https://github.com/atreya2011/go-kratos-test)
```
Comment (atreya2011) [inline docs/ecosystem/community.md:29]: @vinckr 

Thank you for adding this to the docs!
Is it possible to update the link here as well? 🙏🏼 

```suggestion
- [Ory Hydra/Kratos Integration in Go](https://github.com/atreya2011/go-kratos-test/tree/hydra-consent)
```

Comment (vinckr) [reply docs/ecosystem/community.md]: Oh yea I missed that, thanks!

Comment (atreya2011) [reply docs/ecosystem/community.md]: Thank you 🙇🏼

**Review by aeneasr — APPROVED** [review]

**Pull request merged**